docs/src/components/pricing/PricingTable.tsx

```diff
@@ -636,6 +663,16 @@ const rowHeaders: Record<string, React.ReactNode> = {
 const yes = <IconImage name="pricing/yes" title="Included" />;
 const pending = <IconImage name="pricing/time" title="Work in progress" />;
 const no = <IconImage name="pricing/no" title="Not included" />;
+const toBeDefined = (
+  <Typography
+    component="a"
+    href="https://forms.gle/19vN87eBvmXPjBVp6"
+    sx={{ display: 'inline-block' }}
+    title="Take part to this feature shaping"
+  >
+    To define
```

Comment:
"To define" sounds a bit weird, "To be defined" too long. I'd be fine with TBD.
Maybe we can complement it with the info icon that we see in other places on the pricing page.

```suggestion
    TBD
```

Reply:
For me, TBD means "To Be Done" which is the same as pending

Reply:
"to be determined"

But I agree that it's far from obvious

Reply:
Thanks, I put TBD with the extended version in the title